macos/FluentUI/Button/Button.swift

```diff
@@ -352,6 +358,11 @@ open class Button: NSButton {
 	}
 
 	private func setColorValues(forStyle: ButtonStyle, accentColor: NSColor) {
+
+		// Use System Colors which respond correctly to accessibility settings like increase contrast
+		// https://developer.apple.com/documentation/appkit/nscolor/ui_element_colors
+		let increaseContrastBorderColor: NSColor = .textColor
+
```

Comment:
Thoughts on changing this to a private static let on Button so we get implicit laziness? And then each reference to it below would be Button.increaseContrastBorderColor inline.

Advantages:
- trivial perf - not calculated until needed (even though calculation is super cheap now, it might not stay that way as logic gets added.
- simplicity - someone reading through or debugging code doesn't need to rationalize this value unless it is actually needed